cmd/buildtools/openebs.go

```diff
@@ -78,3 +78,115 @@ var updateOpenEBSAddonCommand = &cli.Command{
 		return nil
 	},
 }
+
+type addonComponent struct {
+	name                        string
+	wolfiPackageName            string
+	getWolfiPackageName         func(upstreamVersion string) string
+	upstreamVersionMakefileVar  string
+	upstreamVersionFlagOverride string
+}
+
+var openebsComponents = []addonComponent{
+	{
+		name:                        "openebs-provisioner-localpv",
+		wolfiPackageName:            "dynamic-localpv-provisioner",
+		upstreamVersionMakefileVar:  "OPENEBS_IMAGE_VERSION",
```

Comment:
should we just extract these versions from the charts instead of tracking them in the Makefile? we already have the code for that.